src/private/warehouse.ts

```diff
@@ -67,6 +152,7 @@ export const warehousePrivateHandlers: PrivateNatsHandler[] = [
                 const { start, limit } = (data as unknown) as SearchQuery;
```

Comment:
the public handler passes start and limit as string and not number, so the validation fails and the endpoint won't work. We should convert the strings in the public handler to numbers